fuzz.go

```diff
@@ -0,0 +1,46 @@
+// +build gofuzz
+
+package redis
```

Comment:
I am going to move this go `redis/fuzz` sub-package just in case. Hopefully it does not break anything.